### ISSUE TITLE
Refactor logging

### ISSUE THREAD

**Issue by openjck** [mdn/discord#70]

**Comment by openjck** [comment]
I was thinking we could change the signatures of the functions such that messages are always required, but additional data is optional. Something like this...

```javascript
function log(message, additionalData) {
    console.log('Log: ' + message);
    if(additionalData) {
        console.log(additionalData);
    }
}
```

What do you think @darkwing?

**Comment by openjck** [comment]
Done with #76.

It looks like the commit message didn't auto-close this issue. According to the [documentation](https://help.github.com/articles/closing-issues-via-commit-messages/) commit messages need to use the number sign to auto-close the corresponding issue. For example:

> Fix #70 ...
